src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+public class Todo extends Task {
+    Todo(String desc) {
```

Comment:
Perhaps you could name the argument as 'description'  instead of 'desc' so it is more intuitive, but 'desc' is alright too!